PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -278,17 +278,22 @@ def miniAOD_customizeMC(process):
     process.photonMatch.src = cms.InputTag("reducedEgamma","reducedGedPhotons")
     process.tauMatch.matched = "prunedGenParticles"
     process.tauGenJets.GenParticles = "prunedGenParticles"
+    process.patJetPartons.particles = "prunedGenParticles"
     process.patJetPartonMatch.matched = "prunedGenParticles"
     process.patJetPartonMatch.mcStatus = [ 3, 23 ]
     process.patJetGenJetMatch.matched = "slimmedGenJets"
+    process.patJetPartonsAK8.particles = "prunedGenParticles"
     process.patJetGenJetMatchAK8.matched =  "slimmedGenJetsAK8"
     process.patMuons.embedGenMatch = False
     process.patElectrons.embedGenMatch = False
     process.patPhotons.embedGenMatch = False
     process.patTaus.embedGenMatch = False
     process.patJets.embedGenPartonMatch = False
-    #also jet flavour must be switched to ak4
+    process.patJets.addJetFlavourInfo = True
+    process.patJetsAK8.addJetFlavourInfo = True
+    #also jet flavour must be switched
     process.patJetFlavourAssociation.rParam = 0.4
+    process.patJetFlavourAssociationAK8.rParam = 0.8
```

Comment:
Not needed since it is configured automatically by `addJetCollection()` which takes `rParam=0.8` as one of its input arguments.